testsuite/forge/src/interface/network.rs

```diff
@@ -3,13 +3,23 @@
 
 use super::Test;
 use crate::{CoreContext, Result, Swarm, TestReport};
+use std::{thread, time::Duration};
 
 /// The testing interface which defines a test written with full control over an existing network.
 /// Tests written against this interface will have access to both the Root account as well as the
 /// nodes which comprise the network.
 pub trait NetworkTest: Test {
     /// Executes the test against the given context.
     fn run<'t>(&self, ctx: &mut NetworkContext<'t>) -> Result<()>;
+
+    /// TODO need to implement proper node recovery check:
+    /// endpoint check
+    /// commit seq check
+    /// node health check
+    fn recovery(&self) -> Result<()> {
```

Comment:
This probably doesn't need to be a function on this trait, but maybe on the backend?